src/Symfony/Bundle/TwigBundle/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+7.2
```

Comment:
This PR needs to be rebased on 7.3

Reply:
I already rebased it, but forgot to change the Changelog header, fixed 👍